HamsterListAndroid/build.gradle.kts

```diff
@@ -16,17 +17,14 @@ android {
         minSdk = 26
         targetSdk = 34
         versionCode = 2
-        versionName = "1.1"
+        versionName = "1.1+$buildTag"
     }
     signingConfigs {
         register("app") {
-            val localProperties = Properties().apply {
-                load(project.rootProject.file("local.properties").inputStream())
-            }
-            keyAlias = localProperties.getProperty("keyAlias")
-            keyPassword = localProperties.getProperty("storePassword")
-            storeFile = file(localProperties.getProperty("storeFile"))
-            storePassword = localProperties.getProperty("storePassword")
+            keyAlias = "HamsterListKey"
```

Comment:
Not sure what the key is called. We could also use a CI variable for this.

Reply:
After considering, I don't think the key alias is a secret. I'll add it via commit.